lookout/style/format/benchmarks/release_v0.1/30-seconds-of-code

```diff
@@ -0,0 +1,82 @@
+[1;36mINFO[0m:94f5:QualityReportAnalyzer:[0mClassification report:
```

Comment:
I think we should put only one big joint report as soon as we solve https://github.com/src-d/style-analyzer/issues/313

Reply:
These reports are useful too. It allows seeing more details about each repository and how it will be changing over time.

Reply:
@EgorBu This should be written to a file directly, no logging and stdout redirection.

Reply:
I don't see an easy way to do it because it's a comment generated by `quality report` analyzer. Anyway - it will be replaced with proper markdown report generated with jinja2 help